manifests/backend.pp

```diff
@@ -77,8 +85,10 @@
   Boolean $service_enable = true,
   Stdlib::Absolutepath $state_dir = '/var/lib/sensu/sensu-backend',
   Hash $config_hash = {},
-  Optional[String] $ssl_cert_source = $facts['puppet_hostcert'],
-  Optional[String] $ssl_key_source = $facts['puppet_hostprivkey'],
```

Comment:
Changing this default would be a breaking change. It's possible to make `undef` if the manifest later checks $ssl_*_content and sets $ssl_*_source to the previous defaults if content is undefined.

Reply:
I applied a fix, now when both parameters are empty the source parameters get a default variable